Resources/js/index.js

```diff
@@ -24,19 +23,14 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
-namespace PrestaShop\Module\ExampleModuleMailtheme\Controller\Admin;
+const $ = require('jquery');
+const colorpicker = require('bootstrap-colorpicker/dist/js/bootstrap-colorpicker.min');
 
-use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
-use Symfony\Component\HttpFoundation\Request;
+//This component comes from PrestaShop core, this path is relative so you need to build assets with your modules
+//installed in a PrestaShop project
+import TranslatableInput from '../../../../admin-dev/themes/new-theme/js/components/translatable-input';
```

Comment:
Is it working when admin directory is not admin-dev?

Reply:
I think you need to find a global variable from Prestashop Admin to retrieve the path :thinking:

Reply:
I'm not sur such variable exist in JavaScript 🤔
Besides this is only used during development since is then compiled and only the assets are used in the module. So I figured this was ok (I added the comment to clarify this though)
By the way, our admin core js files work the same way (although less problem for them since they already are in the admin-dev folder)

Reply:
ps_linklist (the only other module using core javascript) only worked with the same principle
But I agree with you we should probable have a way to define the admin folder in javascript, the thing is you would have to store it in a js file, and how do you include it if you don't know its folder?
The very best solution would be to externalize our javascript code in a library so that modules can integrate it, this no matter where the admin folder is AND you don't even need to develop your module inside a PrestaShop instance

Reply:
You're right, I'm not awake ^^'
I'm in favor of making our code library external. It will be harder and harder to use in the future :/